Add Orgs Documentation (#145)
radicle-dev/radicle-docs: docs/connecting-to-ethereum/adding-members.md

---
id: adding-members
title: Adding members
---

On Radicle, peers track projects they are interested in. If a peer wants to
contribute to a project, they fork it, create their own view of the project that

> they publish changes to and then package those changes as a patch.

Reviewer: "they fork it, make their changes and publish them as a patch"

Therefore, to view someone else's patch to your project, you first need to add

> them as a remote to track their view of your project.

Reviewer: ```suggestion
them as a remote to track their changes on your project.
```

### Adding remotes
To add a peer as a remote, you must add their **Device ID** to the list of
remotes that you follow. You can do so by clicking the _Manage Remotes_ option
in the peer selector on your project page. From there, you can add their Device
ID. This is also where you can switch between the different views of peers to
see their changes before fetching them locally.

<video style="border-radius: 16px; margin: 16px 0; width: 100%; height: 100%;"
  playsinline autoplay muted loop>
  <source src="/img/add-remote.webm" type="video/webm">
</video>

> Adding a peer as a remote automatically tracks their view of your project,

Reviewer: ```suggestion
Adding a peer as a remote automatically tracks their project tree,
```

---
id: adding-members
title: Adding members
---

> With multi-signature orgs, you can add multiple **members** who can also initiate transactions such as registrations or anchors in your org. If you created a single-signer org, you can not add more members. Read more about creating an org [here][co].

Reviewer 2: ```suggestion
With multi-signature orgs, you can add multiple **members** who can also initiate transactions such as registrations or anchors in your org.  Read more about creating an org [here][co].

> Note: If you created a single-signer org, you can not add more members.
```

## Adding a member

> To start, navigate to the **Members** tab in your org page.

Reviewer 2: ```suggestion
After creating a multi-signer org, navigate to the **Members** tab in your org page.
```

![single member org page][om]

To add members, you must do so in your Gnosis Safe. Clicking **Add a member** will lead you to your Safe. 

![single member button press][bp]

> If you haven't already, connect your wallet to the Gnosis Safe UI. Be sure to use the one that is linked to your Radicle identity. Click on the **Settings** tab and navigate to *Owners*.

Reviewer 2: ```suggestion
If you haven't already, connect your wallet to the Gnosis Safe UI. Be sure to use the one that is linked to your Radicle identity. Click on the **Settings** tab and navigate to **Owners**.
```

![gnosis safe ui owners][gso]

> Here, you can add additional owners and update your transaction threshold. This is number of owners needed to sign-off on transactions within the Gnosis Safe. This will be the number of people required to sign off on any anchor or other org interaction. Follow the instructions outlined in the [Gnosis Safe docs][gsd] if you need help.

Reviewer 2: ```suggestion
Here, you can add additional owners and update your quorum threshold. This is number of owners needed to sign-off on transactions within the Gnosis Safe. This will be the number of people required to sign off on any anchor or other org interaction. Follow the instructions outlined in the [Gnosis Safe docs][gsd] if you need help.
```
